src/App.tsx

```diff
@@ -1,14 +1,49 @@
 /* eslint-disable max-len */
```

Comment:
Delete `eslint-disable`. Using `eslint-disable` is a bag practice. Try to avoid it here and in test tasks. Find a way to solve it without `eslint-disable`.